runtime/oti/j9nonbuilder.h

```diff
@@ -5799,8 +5799,7 @@ typedef struct J9JavaVM {
 	omrthread_monitor_t tlsFinalizersMutex;
 	struct J9Pool *tlsPool;
 	omrthread_monitor_t tlsPoolMutex;
-	jclass jlThreadConstants;
-	jfieldID vthreadGroupID;
+	j9object_t *vthreadGroup;
```

Comment:
This should really be `jobject`

Reply:
Updated: changed from `j9object_t *` to `jobject`.